README.md

```diff
@@ -10,6 +10,32 @@ The important parts are two functions that assist with setting up and running th
 
 When initializing the new bot, all you have to do is call `build_bot()` and then pass your runtime function into `run_until_dead()`. When `run_until_dead()` returns, everything necessary has been processed to allow the bot to die. Set `run_until_dead()` as the last line under `if __name__ == '__main__'` and the bot will exit cleanly after this returns.
 
+## Example
+
+```python
+from tor_core.initialize import build_bot
+from tor_core.helpers import run_until_dead
```

Comment:
Needs `from tor_core.config import config` in order to interact with the global config object.

Reply:
Oops! Good catch!